### PR TITLE
Update the survey link and text

### PR DESCRIPTION
**Pull request by belbiy**
## This PR is a:

- [ ] New topic
- [ ] Content update
- [X] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will update the cloud survey to have a new link and shorter text.

survey

## Additional information

List all affected URLs:

- Cloud guide only

### CODE REVIEW

File: common/js/survey.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ $(function() {
...
+      var $banner = $('<div class="bs-callout" style="display:none; text-align:left; max-width: 1024px; margin-top: 50px;"><h2>Hello Magento Community!</h2><p>Please share your feedback through this quick 5 minute survey.</p><p>Thank you in advance for participating. Your feedback is 100% confidential and will not be shared outside of Magento.</p> <button data-href="https://www.surveymonkey.com/r/CloudFeatures-C" class="btn btn-primary btn-large">OK, I will help</button> <button class="btn btn-large">No, thanks</button> </div>');
```
Comment (meker12) [inline common/js/survey.js]:
```suggestion
      var $banner = $('<div class="bs-callout" style="display:none; text-align:left; max-width: 1024px; margin-top: 50px;"><h2>Hello Magento Community!</h2><p>Please share your feedback by taking this quick 5 minute survey.</p><p>Thank you in advance for participating. Your feedback is 100% confidential and will not be shared outside of Magento.</p> <button data-href="https://www.surveymonkey.com/r/CloudFeatures-C" class="btn btn-primary btn-large">OK, I will help</button> <button class="btn btn-large">No, thanks</button> </div>');
```

**Comment by belbiy** [comment]
running tests

**Comment by belbiy** [comment]
running tests

**Comment by belbiy** [comment]
@meker12 thank you for the feedback on the message text!

**Pull request merged**